src/main/java/com/marginallyclever/makelangelo/CollapsiblePanel.java

```diff
@@ -127,17 +127,17 @@ protected void toggleVisibility(boolean visible) {
         }
         updateBorderTitle();
         if (visible) {
-            Dimension toggle = new Dimension(parentFrame.getWidth(), previousDimension.height);
-            parentFrame.setPreferredSize(toggle);
+            Dimension toggle = new Dimension(parentWindow.getWidth(), previousDimension.height);
+            parentWindow.setPreferredSize(toggle);
         } else {
-            previousDimension = parentFrame.getSize();
+            previousDimension = parentWindow.getSize();
             int height = previousDimension.height - innerPannel.getHeight();
             Dimension toggle = new Dimension(previousDimension.width, height);
-            parentFrame.setPreferredSize(toggle);
+            parentWindow.setPreferredSize(toggle);
         }
-        parentFrame.validate();
-        parentFrame.repaint();
-        parentFrame.pack();
+        parentWindow.validate();
```

Comment:
revalidating this item cause the parent to automatically revalidate?  These may be overreach.

Reply:
I used it order to redraw the parent interface when the panel is collapsed. otherwise, the parent window stays at the maximum height